### PR TITLE
[ BLAS ] use openblas function to set num threads

### PR DESCRIPTION
**Pull request by jijoongmoon**
This patch use openblas_set_num_thread() to set the number of thread
for BLAS computation, instread of set environment variables

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nntrainer/tensor/blas_interface.cpp [hunk 1]
```diff
@@ -16,6 +16,10 @@
 
 #include <cmath>
 
+#if (BLAS_NUM_THREADS > 0) && (BLAS_NUM_THREADS < 9999)
+#define NUM_THREADS_ BLAS_NUM_THREADS
```
Comment (lhs8928) [inline nntrainer/tensor/blas_interface.cpp:20]: I wonder do we need to define NUM_THREADS_ instead of directly use BLAS_NUM_THREADS?

**Review by jihochu — APPROVED** [review]
Looks good!

File: meson.build [hunk 2]
```diff
@@ -152,8 +154,23 @@ if get_option('enable-blas')
     # for Ubuntu 20.04
     if not blas_dep.found()
       blas_dep = dependency('openblas')
+      blas_version = blas_dep.get_pkgconfig_variable('version').split('.')
+    else
+       blas_version = blas_dep.version().split('.')
```
[the remark this answers is not in the document]
Comment (jijoongmoon) [reply meson.build]: fixed. thanks.

File: nntrainer/tensor/blas_interface.cpp (hunk 1 above)
Comment (jijoongmoon) [reply nntrainer/tensor/blas_interface.cpp]: done. thanks.

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**